fix issue with Intel compilers: 'the object has cv-qualifiers that are not compatible with the member function'
author: Kenneth Hoste (Ghent University)
--- /tmp/vsc40023/easybuild_build/SaguaroGW/20150315/intel-2015a/saguarogw-code-29/base/CommandLineParser.h.orig	2012-05-16 15:40:12.000000000 +0200
+++ /tmp/vsc40023/easybuild_build/SaguaroGW/20150315/intel-2015a/saguarogw-code-29/base/CommandLineParser.h	2015-05-12 21:54:40.830166431 +0200
@@ -39,8 +39,8 @@
 string GetName() { return mName; }
 string GetDescription() { return mDesc; }
 
-friend bool operator< (const commandArg<argType> &lhs, 
-		       const commandArg<argType> &rhs)
+friend bool operator< (commandArg<argType> &lhs, 
+		       commandArg<argType> &rhs)
 {
   if ( lhs.GetName() < rhs.GetName() )
     return true;
